### DISCUSSION TITLE
lineage data is not ready

### DISCUSSION THREAD

**Discussion by jackysrivastava** [AbsaOSS/spline-getting-started#53] [Q&A]
Afer submiting the pyspark code, I'm not able to see the data lineage under the execution plan in Spline UI. It's showing the lineage data is not available. Please help to resolve the issue.

**Comment by cerveada** [comment]
Hello, 
try to do steps from this troubleshooting guide: https://github.com/AbsaOSS/spline/discussions/1225#discussioncomment-5857789
